js/components/generate/GenerateForm.js

```diff
@@ -55,8 +58,8 @@ export default class GenerateForm extends React.Component {
           setErrors({
             courses:
               missingCourses.length === 1
-                ? `The course ${missingCourses} was invalid! Please check your input.`
-                : `The courses [${missingCourses.join(", ")}] were invalid! Please check your input.`,
+                ? `Unrecognized course ID: ${missingCourses}`
```

Comment:
Reword to "Invalid course code: ..." (and "codes" for the plural version of the message)